### ISSUE TITLE
How to clear selected options?

### ISSUE THREAD

**Issue by kastureranjit** [NileshPatel17/ng-multiselect-dropdown#65]
I have 3 dropdowns each depends upon another. When I unselect all then next drop-down is still showing the previously selected data. So how to clear the data?

**Comment by NileshPatel17** [comment]
please share sample code using codesandbox or https://stackblitz.com, otherwise very difficult to help you

**Comment by viesca03** [comment]
Hi, I have the same issue here and I can't find the solution. Do you have any update or solution for this?

Thanks in advance.

**Comment by MongiSlimen** [comment]
Hello @viesca03  @kastureranjit  @NileshPatel17 the same issue here there is any solution ?

**Comment by NileshPatel17** [comment]
@MongiSlimen 	please share sample code, i will help you.

**Comment by MongiSlimen** [comment]
it's resolved for me i just explained how.

**Comment by quiters89** [comment]
@MongiSlimen  yeah the data is cleared. how about the UI side how did you remove the selected display in the UI? thanks,

**Comment by viesca03** [comment]
Thanks for the answer @MongiSlimen i eill try that tomorrow! Thanks a lot 😁

**Comment by MongiSlimen** [comment]
@quiters89  i did an exemple for you https://stackblitz.com/edit/angular-x5pqvu?file=src%2Fapp%2Fapp.component.ts

**Comment by NileshPatel17** [comment]
@quiters89 	check it out below
https://stackblitz.com/edit/angular-dnmu2a

**Comment by quiters89** [comment]
does the version 0.1.4 support this @NileshPatel17 ?

Thank you sooo much for the demo files. really means a lot. all your efforts. someday good karma will comeback to you. @MongiSlimen  @NileshPatel17  thank you again for the support.

**Comment by vinayrevanna** [comment]
How to reset some of the selected option in dependent options and retain other selections?
